### DISCUSSION TITLE
Handling non-literal C-style escapes

### DISCUSSION THREAD

**Discussion by J-Kappes** [jarun/nnn#1739] [Q&A]
Assuming
```nu
let str = `foo\'bar`
```
is there a way to get `foo'bar` out of `str` analogously to the output of `"foo\'bar"`? `$"($str)"` doesn't work.

**Comment by 0xACE** [comment]
what language are you working with? how are you trying to use this with `nnn`?

**Comment by J-Kappes** [comment]
Apologies, wrong repo.

The question is about the nushell language.

**Reply by N-R-K** [reply]
If you get around asking this in nu-shell repo/forum then try to ask about the [actual problem](https://github.com/jarun/nnn/pull/1737#issuecomment-1735351494) instead of the attempted solution. They likely know a better way to solve it, see: [xyproblem](https://xyproblem.info/).
